src/components/Input/Input.tsx

```diff
@@ -0,0 +1,339 @@
+/**
+ * Copyright 2019, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC, HTMLProps, ReactNode } from 'react';
```

Comment:
FC was removed from [cra`s template](https://github.com/facebook/create-react-app/pull/8177). I don't think we need to refactor this bit to remove it, but the thoughts are interesting nonetheless

Reply:
Interesting indeed! I started using it when I was new to TypeScript. The reasoning in the issue you linked makes sense. I think we should stop using it and remove it from Circuit UI and Foundry.

Reply:
Done in https://github.com/sumup-oss/foundry/pull/115.